include/deal.II/base/qprojector.h

```diff
@@ -322,6 +350,16 @@ class QProjector
          const bool                      face_rotation,
          const hp::QCollection<dim - 1> &quadrature);
 
+    /**
+     * Like the above function but taking a quadrature collection, enabling
+     * that each face might have different number of quadrature points.
```

Comment:
I am not entirely sure, but would
```suggestion
     * that each face might have a different number of quadrature points.
```
be slightly more accurate?

Reply:
You're right. I find the comments of these functions pretty hard to understand - I'll add rewriting them to the orientation TODO list.

For now I rewrote this comment and the equivalent one for the deprecated version.